lib/notifications/use_case/send_confirmation_email.rb

```diff
@@ -0,0 +1,28 @@
+module Notifications
```

Comment:
I quite like the `Actor::UseCase::Thingy` naming convention - wondering if there's something clearer than `Notifications` to be the top-level for this one?

I say this, and I couldn't find something better than `Administrator` for a previous feature.

... maybe this one is `Administrator` too? Or maybe it's `Common`, if "notifications" is a thing the app does for multiple actors?